pkgs/top-level/all-packages.nix

```diff
@@ -2021,6 +2021,8 @@ in
 
   jellyfin = callPackage ../servers/jellyfin { };
 
+  jellyfin_10_5 = callPackage ../servers/jellyfin/10.5.x.nix { }
```

Comment:
```suggestion
  jellyfin_10_5 = callPackage ../servers/jellyfin/10.5.x.nix { };
```

:stuck_out_tongue:

Reply:
Sigh... pushing.
On a side note, how do you call nix-review?

Reply:
From your side, you should be able to call `nix-review wip`, or `nix run nixpkgs.nix-review -c nix-review wip` if you haven't installed it.

Reply:
If you want to review a PR, it's similar: `nix-review pr 93654`

Reply:
Yeah wip didn't work, but `pr` did.